order-service/src/main/java/com/microcommerce/orderservice/service/OrderService.java

```diff
@@ -81,13 +74,20 @@ public ApiResponse<OrderResponse> createOrder(OrderRequest orderRequest) {
                 .build();
 
         orderRepository.save(orderEntity);
-        produceOrderCreatedEvent(orderCreatedEvent);
+        produceOrderCreatedEvent(orderItems, orderNumber, orderRequest.getUser());
 
         return responseFactory.successfulOperation();
     }
 
-    private void produceOrderCreatedEvent(OrderCreatedEvent orderCreatedEvent) {
+    private void produceOrderCreatedEvent(List<EventOrderItemDto> orderItems, String orderNumber, UserDto user) {
+        var orderCreatedEvent = OrderCreatedEvent.builder()
+                .orderItems(orderItems)
+                .orderNumber(orderNumber)
+                .user(user)
+                .build();
+
         var completableFuture = kafkaTemplate.send("notification-topic", UUID.randomUUID(), orderCreatedEvent);
```

Comment:
Define this configuration value in application yaml file.

Reply:
Fixed